### PR TITLE
virtio: dom0: add 'no GPU, no display' config for DomU

### PR DESCRIPTION
**Pull request by rshym**
This commit adds possibility to use DomU without
any display and 3D GSX.
This is part of the preparation for the console-only set up.

### CODE REVIEW

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend [hunk 1]
```diff
@@ -37,7 +38,11 @@ do_install:append() {
 
     if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'true', 'false', d)}; then
         cat ${WORKDIR}/virtio.cfg >> ${CFG_FILE}
-        cat ${WORKDIR}/domu-vdevices-virtio.cfg >> ${CFG_FILE}
+        if ${@bb.utils.contains('MACHINE_FEATURES', 'gsx', 'true', 'false', d)}; then
+            cat ${WORKDIR}/domu-vdevices-virtio.cfg >> ${CFG_FILE}
```
Comment (lorc) [inline meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:42]: This is getting out of hand. We really need come .bbclass to generate those configs dynamically.

This is just a note... no actions need to be taken right now.

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend (hunk 1 above)
Comment (rshym) [reply meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:42]: Fully agree.

**Comment by rshym** [comment]
`Ack` is added. Successfully built and ran on h3ulcb-4x2g with `console` and `sources`.
Ready to be merged.

**Pull request merged**